### PR TITLE
gimp: improve grammar

### PR DESCRIPTION
**Pull request by sbrl**
The edit to the description is optional, but the other edits are grammar improvements.

I saw that this page had already been merged before I got to it, so I opened this new PR instead of commenting there.

### CODE REVIEW

**Comment by bl-ue** [comment]
>I saw that this page had already been merged before I got to it, so I opened this new PR instead of commenting there.

If we get any more collaborators we may need to raise the minimum approval count 🤔

**Review by bl-ue — APPROVED** [review]

**Comment by sbrl** [comment]
@bl-ue generally I'd suggest leaving PRs open at least 2 days to give people a chance to comment on it.

@tldr-bot, you have *no* idea what you're talking about.

@owenvoke, I think we have a bit of a problem with the linter here.

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
It's the `Always` [see here](https://github.com/tldr-pages/tldr-lint/blob/81cf0c8d342d0faa3d329cbc64ec8f7779bb7c76/tldr.l#L154-L156):
```y
if (yytext.match(/(^[A-Za-z]{3,}ing )|(^[A-Za-z]+[^us]s )/)) {
    yy.error(yylloc, 'TLDR104');
}
```

File: pages/common/gimp.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-> An image manipulation and paint program.
+> GNU image manipulation program.
 > More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
 
 - Launch GIMP:
 
 `gimp`
 
-- Launch but don't show the splash screen:
+- Launch, but don't show the splash screen:
```
Comment (waldyrious) [inline pages/common/gimp.md:10]: When possible, I'd suggest avoiding context-sensitive example descriptions (i.e. relying on the sentence structure of the previous example's description):

```suggestion
- Launch GIMP without showing the splash screen:
```

File: pages/common/gimp.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
 `gimp --no-splash`
 
-- Start a new instance, even if there is already a running one:
+- Always start a new instance:
```
Comment (waldyrious) [inline pages/common/gimp.md:14]: In light of my previous comment above, I'd be inclined to actually double down on the existing description:

```suggestion
- Start a new GIMP instance, even if there is already a running one:
```

File: pages/common/gimp.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/gimp.md:10]: Hmm...we do that in a lot of pages though.  I actually rely on it if the previous description is lengthy ;)

Comment (waldyrious) [reply pages/common/gimp.md:10]: Exactly, it makes sense as a trade-off to balance the length of the description. In this case, it doesn't seem to be necessary, so there's only the downside of the reliance on another description.

Comment (bl-ue) [reply pages/common/gimp.md:10]: Ah, I see what you're saying, then.  Yes, that does make _perfect_ sense.

**Comment by bl-ue** [comment]
The unfortunate thing about this is that now we have to go through one of those annoying linter update sagas—but now we can use the `stalebot-ignore` label 😉

File: pages/common/gimp.md (hunk 1 above)
Comment (sbrl) [reply pages/common/gimp.md:10]: Great idea! Let's do this.

**Review by owenvoke — APPROVED** [review]

**Comment by bl-ue** [comment]
So the CI checks are all green 🤔

**Review by bl-ue — APPROVED** [review]

**Pull request merged**